AUTHORS.md

```diff
@@ -0,0 +1,26 @@
+goamz Authors
+=============
+
+This file contains the list of the official goamz maintainers.
```

Comment:
The official maintainers and the list of authors is not actually the same. It also seems unnecessary to have a list of official maintainers.. the point of contact should be the project itself and the mailing list. The maintainers should be watching those.

I suggest "dumbing down" this file to be a list of authors, people that contributed relevant code to the project.